src/elements/expansion-panel/expansion-panel-header/expansion-panel-header.visual.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { html, nothing } from 'lit';
+
+import {
+  describeEach,
+  describeViewports,
+  visualDiffStandardStates,
+  visualRegressionFixture,
+} from '../../core/testing/private.js';
+
+import './expansion-panel-header.js';
+import '../../icon.js';
+
+describe(`sbb-expansion-panel-header`, () => {
+  let root: HTMLElement;
+
+  const cases = {
+    disabled: [false, true],
+    state: [
+      { icon: undefined, slotted: false },
+      { icon: 'arrow-right-small', slotted: true },
+      { icon: 'arrow-right-small', slotted: false },
+    ],
+  };
+
+  describeViewports({ viewports: ['zero', 'medium'] }, () => {
+    describeEach(cases, ({ disabled, state }) => {
+      beforeEach(async function () {
+        root = await visualRegressionFixture(html`
+          <sbb-expansion-panel-header
+            ?disabled=${disabled}
+            .iconName=${!state.slotted ? state.icon : undefined}
+          >
+            ${state.slotted ? html`<sbb-icon name=${state.icon!}></sbb-icon>` : nothing} Label
+          </sbb-expansion-panel-header>
+        `);
+      });
+
+      for (const state of visualDiffStandardStates) {
```

Comment:
In theory there is not active state, so you could also use a subarray without active state. But it's also not so bad. Moreover, wouldn't it be sufficient that this component is already tested in expansion-panel? Or do the tests here provide any additional value? e.g. the focus state does not look correctly when not using inside expansion panel.

Reply:
I wanted to separate the test responsibilities but maybe that's not the way to go. 
In theory, separating them helps to troubleshoot (you immediately know what component has it). Do you have an opinion?
Ps. I removed the focus tests (the expansion-panel is responsible for it)

Reply:
Basically, I agree with you - we should test at specific component level :) We can also start like you proposed for now. 
In general, I try to avoid too much noise by e.g. flickering tests (The more tests the more potential problems). And on the other hand, the chance that if in the expansion-header there is a failing test but in expansion-panel tests not is very low, I think. So in most (all?) cases, both will fail. So we would have more diffs to check.

Reply:
That's also true :/
Ideally, we should try not to overlap the test cases as much as possible.
As you said, the only exclusive cases for the panel-header are the icon combinations
I'm convinced, I'll merge them with the expansion panel